examples/multi-sheet.ts

```diff
@@ -0,0 +1,29 @@
+// workbook example shows how to structure with multiple imported sheets
+import { Portal, Workbook} from '@flatfile/configure'
+
+// Define your own sheets in seperate files for easier maintainability
```

Comment:
I would put the multi-sheet example in it's own directory so we can point a user at "here's our whole multi file example directory tree"

Reply:
I like it - and will focus this PR on this type of `/multiSheet/` example